modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -31,7 +32,7 @@ class UploadsController < ApplicationController
       def submit
         Datadog::Tracing.active_trace&.set_tag('form_id', params[:form_number])
 
-        response = if form_is210966 && icn && first_party?
+        response = if form_is210966 && first_party?
```

Comment:
I've removed `icn` here because I think it is redundant if we've already done the `mpi_proxy` business.